docs/samples/browser-plugin-meetings/app.js

```diff
@@ -2694,6 +2694,39 @@ function viewBreakouts(event) {
 
     return button;
   };
+  const createAskForHelpButton = (breakoutSession) => {
+    const button = document.createElement('button');
+
+    button.innerText = 'Help';
+
+    button.onclick = () => {
+      breakoutSession.askForHelp();
+    };
+
+    return button;
+  };
+  const createAskAllReturnButton = (breakoutSession) => {
+    const button = document.createElement('button');
+
+    button.innerText = 'Return';
+
+    button.onclick = () => {
+      meeting.breakouts.askAllToReturn();
+    };
+
+    return button;
+  };
+  const createBroadcastButton = (breakoutSession) => {
+    const button = document.createElement('button');
+
+    button.innerText = 'broadcast';
+
+    button.onclick = () => {
```

Comment:
suggestion: It would be nice if it were possible to test the cohost/presenter logic in the sample app. Perhaps tickboxes for those options?

Reply:
![breakouts_dropdown](https://user-images.githubusercontent.com/3998548/220586087-0ff2fdb6-d495-42dd-baf1-f9f9cb836077.PNG)

At the moment the broadcast button just sends a broadcast message to all the sessions and all the participants. If you see the above screenshot, we are unable to test sending to presenters/cohosts/hosts in the different combinations from the sample app. We are also unable to test sending a message to a particular breakout session

Reply:
Got it. 👍